lib/collection/header.js

```diff
@@ -190,6 +192,38 @@ _.extend(Header, /** @lends Header */ {
         var args = Array.prototype.slice.call(arguments);
         args.unshift(Header);
         return new (Header.bind.apply(Header, args))();
+    },
+
+    /**
+     * Gets value from an Array or {@link PropertyList} of Headers by filtering through key.
+     *
+     * @param {Array|PropertyList<Header>} headers
+     * @param {string} key
+     * @returns {string}
+     */
+    getHeaderValue: function(headers, key) {
+        if (!_.isArray(headers) && !PropertyList.isPropertyList(headers)) {
+            throw new Error('It is only possible to unparse PropertyList or Array of headers.');
+        }
+        var header = _.find(headers, function (header) {
```

Comment:
In case you have a PropertyList, you should use http://www.postmanlabs.com/postman-collection/PropertyList.html#find